libkbfs/tlf_journal.go

```diff
@@ -511,6 +515,30 @@ func (j *tlfJournal) flushBlockEntries(ctx context.Context) (int, error) {
 	return entries.length(), nil
 }
 
+func (j *tlfJournal) getNextMDEntryToFlush(ctx context.Context,
+	currentUID keybase1.UID, currentVerifyingKey VerifyingKey) (
+	MdID, *RootMetadataSigned, error) {
+	j.journalLock.RLock()
+	defer j.journalLock.RUnlock()
+	return j.mdJournal.getNextEntryToFlush(
+		ctx, currentUID, currentVerifyingKey, j.config.Crypto())
```

Comment:
Seems like to be symmetric with the block logic, you should be passing in `end` here.